fe1-web/src/core/keypair/__mocks__/KeyPairStore.ts

```diff
@@ -0,0 +1,19 @@
+import { jest } from '@jest/globals';
+
+import { mockKeyPair } from '__tests__/utils';
+
+import { KeyPair } from '../../objects';
+
+let keyPair: KeyPair = mockKeyPair;
+
+export namespace KeyPairStore {
+  export const store = jest.fn<void, [KeyPair]>().mockImplementation((kp: KeyPair) => {
+    keyPair = kp;
+  });
```

Comment:
Can't you write it like this? I feel like it's clearer and less verbose. Not sure about it though. 
```suggestion
  export const store = jest.fn<void, [KeyPair]>((kp: KeyPair) => {
    keyPair = kp;
  });
```
The same would apply to the function below and to every function where you called `.mockImplementation()`.

Reply:
Oh, indeed this is much nicer! I did not know about this, thank you! 😁